ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json

```diff
@@ -478,7 +478,7 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseEventID": "uploadOtherCourtAdminDocuments-PREPARE_FOR_HEARING",
+    "CaseEventID": "uploadOtherCourtAdminDocuments",
```

Comment:
Since we are deprecating event and having new ID, maybe there is a better name? manageDocumentsAfterGatekeeping?

Reply:
renamed to manageDocuments, eventually lets deprecate uploadStandardDirections (if business agree that uploading SDO should exist elsewhere) and use this as the main event